packages/charts/src/components/BarChart/components/Axes.tsx

```diff
@@ -38,7 +38,7 @@ export const Axes = ({
     valueFormatter,
     labelFormatter,
 }: AxesProps) => {
-    const formatLabel = (label: string) => truncateTextLabel(labelFormatter(label));
+    const formatLabel = (label: string) => truncateTextLabel(labelFormatter(label).toString());
```

Comment:
Why is this necessary? 

The typing of laybelFormatter is `export type LabelFormatter = (label: string) => string;` the value is already a string

Reply:
Fixed the typing, the `formatLabel` param is of type `string | number` in reality because of the axis toggle